iris/tests/download_manager/open_download_folder.py

```diff
@@ -48,4 +60,5 @@ def run(self):
 
     def teardown(self):
         # Cleanup.
-        downloads_cleanup('5MB.zip')
+        path = IrisCore.get_downloads_dir()
+        downloads_cleanup(path)
```

Comment:
download_cleanup() function shouldn't have a path parameter. The name is self-explanatory that it refers to the downloads directory